Update readme on when images get built (#1294)
dotnet/dotnet-buildtools-prereqs-docker: README.md

- The corresponding Dockerfile is modified.
- The base image is updated (a new version of the image referred to by the [`FROM`](https://docs.docker.com/engine/reference/builder/#from) statement).

> - Once a month per the schedule defined in the [pipeline](eng/pipelines/dotnet-buildtools-prereqs-all.yml).

Reviewer: Consider leaving this blank so that the documentation has less of a chance of getting stale?

```suggestion
- According to the schedule defined in the [pipeline yaml](eng/pipelines/dotnet-buildtools-prereqs-all.yml).
```
